scripts/replace-placeholders.sh

```diff
@@ -0,0 +1,5 @@
+# Find and replace placeholder values with runtime-specific environment values
+find apps/web/.next/ apps/web/public -type f |
+while read file; do
+    sed -i "s|https://NEXT_PUBLIC_WEBAPP_URL_PLACEHOLDER|$NEXT_PUBLIC_WEBAPP_URL|g" "$file"
```

Comment:
placeholder base should be http to support local dev and match other examples
```suggestion
    sed -i "s|http://NEXT_PUBLIC_WEBAPP_URL_PLACEHOLDER|$NEXT_PUBLIC_WEBAPP_URL|g" "$file"
```

Reply:
Updated.